processing/data_collection/gazette/pipelines.py

```diff
@@ -14,14 +13,6 @@
 from gazette.settings import FILES_STORE
 
 
-class GazetteDateFilteringPipeline:
```

Comment:
I wouldn't remove this pipeline. This will always guarantee that we will return only the desired data, even if any other thing we include (like a Middleware or spider logic) fails.

Reply:
That was removed because it was basically doing the same thing as the spider middleware for the items